src/components/InputSection.jsx

```diff
@@ -22,8 +21,8 @@ function InputSection({
     <main>
       <section className="bg-[url('/src/img/input-section-bg.png')] bg-cover bg-center w-full h-screen border-y border-secondary-color">
         <Wrapper>
-          <div className="relative h-screen flex flex-col md:flex-row justify-center items-center">
-            <div className="md:grow text-sm md:text-base flex flex-col items-center space-y-1 text-center text-primary-dark">
+          <div className="md:grow text-sm md:text-base flex flex-row items-center space-y-1 text-center text-primary-dark mt-80">
```

Comment:
One trick to making this particular mobile responsive would be to introduce a few more classes in this div and also remove a couple. First go ahead and remove `flex`, `flex-row`.
Now add the class of `block` to set the display to block on all screens.
Next would be `lg:flex` to set the display to flex on larger screens
Finally add `w-full` to set the width to 100%.


This will make it responsive but a little bit messy on smaller screens.

Reply:
Why would it be messy on smaller screens?